README.md

```diff
@@ -1,95 +1,6 @@
 # Aframe
-Detecting compact binary mergers from gravitational wave strain data using neural networks, with an emphasis on
-- **Efficiency** - making effective use of accelerated hardware like GPUs in order to minimize time-to-solution
-- **Scale** - validating hypotheses on large volumes of data to obtain high-confidence estimates of model performance
-- **Flexibility** - modularizing functionality to expose various levels of abstraction and make implementing new ideas simple
-- **Physics first** - taking advantage of the rich priors available in GW physics to build robust models and evaluate them accoring to 
-meaningful metrics
-- **Multi-messenger astronomy** - making algorithmic decisions and optimizations that allow for extremely low-latency alerts 
+Detecting compact binary mergers from gravitational wave strain data using neural networks. 

```

Comment:
Can we put a link to the documentation in the README? Or does the page not exist until this PR gets in?

Reply:
Yes. Will make a followup PR that does this (and addresses your below issues) once the build is successful